op/model.py

```diff
@@ -287,9 +314,9 @@ def relation_list(self, relation_id):
             else:
                 raise
 
-    def relation_get(self, relation_id, member_name):
+    def relation_get(self, relation_id, member_name, app):
         try:
-            return self._run('relation-get', '-r', str(relation_id), '-', member_name)
+            return self._run('relation-get', '-r', str(relation_id), '-', member_name, f'--app={app}')
```

Comment:
Does it actually accept _"True"_ and _"False"_ as parameter values?

Reply:
Yes, I checked prior to implementing:

```
relation-set x=1 --app=True ; echo $?
0
```

Reply:
We might have a bug in Juju itself though. For app myapp, unit myunit/0:

`relation-get - myapp`

Returns `ERROR permission denied` on 2.7 RC4.

I will do another check on a newer RC.